cloud/sample.env

```diff
@@ -49,6 +49,16 @@ HYPERDX_APP_URL=http://hyperdx.your-domain.com
 
 # enables logs and metrics saving into clickhouse
 CLICKHOUSE_ENABLED=true
+CLICKHOUSE_URL=http://clickhouse:8123
+
+# clickhouse admin credentials
+CLICKHOUSE_USER=truser
```

Comment:
I remember now that we wanted to look into generating these programmatically and then setting all these directly in mongodb. Do I remember that correctly?